epp-client/src/common.rs

```diff
@@ -98,7 +98,7 @@ impl Options {
 #[serde(rename = "extension")]
 pub struct Extension<E: ElementName> {
     /// Data under the &lt;extension&gt; tag
-    #[serde(alias = "upData")]
+    #[serde(alias = "upData", alias = "namestoreExt")]
```

Comment:
This is still required. I'd like to move away from this but I'm not sure how to handle this yet.